orttraining/orttraining/test/gradient/gradient_ops_test.cc

```diff
@@ -1,7 +1,7 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-#ifdef NDEBUG  // disable for debug builds because some of these tests are slow
+#ifndef NDEBUG  // disable for debug builds because some of these tests are slow
```

Comment:
>ifndef [](start = 1, length = 6)

pls don't forget to change it back

Reply:
You're totally right. I forgot to change it back. It's fixed now